sdk/metricsadvisor/Azure.AI.MetricsAdvisor/tests/MetricsAdvisorAdministrationClient/AnomalyAlertConfigurationLiveTests.cs

```diff
@@ -931,7 +931,8 @@ public async Task DeleteAlertConfiguration(bool useTokenCredential)
 
             try
             {
-                configId = await adminClient.CreateAlertConfigurationAsync(configToCreate);
+                AnomalyAlertConfiguration createdConfig = await adminClient.CreateAlertConfigurationAsync(configToCreate);
```

Comment:
is there a way to have a test where the Get operation fails just to see how it feels?

Reply:
We may be able to do it with mocking and the help of a cancellation token. We have a strong time constraint for the beta 3 release, so I created an issue to track it: https://github.com/Azure/azure-sdk-for-net/issues/18589.